scenarios/examples/vulnerability_examples/shepherd.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0"?>
```

Comment:
Thanks, can you please confirm you've tested the scenario

Reply:
Been using this scenario to test with so, fairly certain once im done with the rest of the review, it should be ok.

Reply:
Just run it again, to test before you let me know it's ready to review